src/30_Advanced/Image_Galleries_with_amp-carousel.html

```diff
@@ -144,7 +144,7 @@
   attribute in the `<p>` tag above the carousel to update as well.
   -->
   <div>
-    <p> Selected slide: <span [text]="(selectedSlide + 1)">1</span>/3</p>
+    <p> Selected slide: <span [text]="(selectedSlide * 1 + 1)">1</span>/3</p>
```

Comment:
We should explain this. Out of curiosity, does this work: `+ selectedSlide + 1`? Might be a bit more idiomatic.